### PR TITLE
Add, and test, account.reject method.

### PR DESCRIPTION
**Pull request by gregsabo**
For the [new API endpoint](https://stripe.com/docs/api/java#reject_account).

### CODE REVIEW

File: src/test/java/com/stripe/model/AccountTest.java [hunk 1]
```diff
@@ -110,6 +110,22 @@ public void testRetrieveAccountWithId() throws StripeException {
...
+		verifyGet(Account.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C");
+
+		Map<String, Object> rejectParams = new HashMap<String, Object>();
+		rejectParams.put("reason", "fraud");
+		acc.reject(rejectParams);
+		verifyPost(Account.class, "https://api.stripe.com/v1/accounts/acct_1032D82eZvKYlo2C/reject", rejectParams);
+
+		verifyNoMoreInteractions(networkMock);
+  }
+
```
Comment (brandur) [inline src/test/java/com/stripe/model/AccountTest.java]: Sorry for the nit, but mind just tightening the whitespace here? Thanks!

**Comment by brandur** [comment]
Left a really minor comment, but looks great. Thanks @gregsabo!

File: src/test/java/com/stripe/model/AccountTest.java (hunk 1 above)
Comment (gregsabo) [inline src/test/java/com/stripe/model/AccountTest.java]: No prob - done! (I think)

**Comment by brandur** [comment]
Released as 1.46.0.

**Pull request merged**